loadtest/tests.go

```diff
@@ -280,14 +280,20 @@ func actionGetChannel(c *EntityConfig) {
 				}
 			}
 
-			if rand.Float64() < c.LoadTestConfig.UserEntitiesConfiguration.CustomEmojiChance {
+			if rand.Float64() < c.LoadTestConfig.UserEntitiesConfiguration.CustomEmojiChance && c.LoadTestConfig.LoadtestEnviromentConfig.NumEmoji > 0 {
 				name := c.LoadTestConfig.LoadtestEnviromentConfig.PickEmoji()
 				if _, resp := c.Client.GetEmojiByName(name); resp.Error != nil {
 					mlog.Error("Unable to get emoji.", mlog.String("emoji_name", name), mlog.String("user_id", post.UserId), mlog.Err(resp.Error))
 				}
 			}
 		}
 	}
+	if rand.Float64() < c.LoadTestConfig.UserEntitiesConfiguration.NeedsProfilesChance {
```

Comment:
In practice, is it just the one user that needs to be resolved? Or to ask this another way, should we sometimes loadtest querying for multiple users? I'm imagining a loop here that accumulates usernames, with the probability of starting an iteration still governed by the `NeedsProfilesChance`.

Reply:
Also, based on my investigation into https://mattermost.atlassian.net/browse/MM-10483, it looks like I'll be adding similar code to cover `/api/v4/users/ids` and `/api/v4/users/status/ids`, so figuring out the pattern here will be tremendously helpful :)

Reply:
Ah, ok, so the average number called by a single entity? My concern is that doesn't really reflect the client behaviour which might load a bunch of these in quick succession vs. distributing the queries out over the average.